packages/stargate/src/queries/ibc.spec.ts

```diff
@@ -14,220 +14,562 @@ async function makeClientWithIbc(rpcUrl: string): Promise<[QueryClient & IbcExte
+          expect(response.channels).toEqual([ibcTest.identifiedChannel]);
+          expect(response.pagination).toBeDefined();
+          expect(response.height).toBeDefined();
 
-        tmClient.disconnect();
+          tmClient.disconnect();
+        });
       });
-    });
 
-    describe("packetCommitment", () => {
-      it("works", async () => {
-        pendingWithoutSimapp();
-        const [client, tmClient] = await makeClientWithIbc(simapp.tendermintUrl);
+      describe("allConnectionChannels", () => {
+        it("works", async () => {
+          pendingWithoutSimapp();
+          const [client, tmClient] = await makeClientWithIbc(simapp.tendermintUrl);
 
-        const response = await client.ibc.unverified.packetCommitment(
-          ibcTest.portId,
-          ibcTest.channelId,
-          ibcTest.commitment.sequence,
-        );
-        expect(response.commitment).toEqual(ibcTest.commitment.data);
-        expect(response.proofHeight).toBeDefined();
-        expect(response.proofHeight).not.toBeNull();
+          const response = await client.ibc.unverified.channel.allConnectionChannels(ibcTest.connectionId);
+          expect(response.channels).toEqual([ibcTest.identifiedChannel]);
 
-        tmClient.disconnect();
+          tmClient.disconnect();
+        });
       });
-    });
 
-    describe("packetCommitments", () => {
-      it("works", async () => {
-        pendingWithoutSimapp();
-        const [client, tmClient] = await makeClientWithIbc(simapp.tendermintUrl);
+      describe("clientState", () => {
+        it("works", async () => {
+          pendingWithoutSimapp();
+          const [client, tmClient] = await makeClientWithIbc(simapp.tendermintUrl);
+
+          const response = await client.ibc.unverified.channel.clientState(ibcTest.portId, ibcTest.channelId);
+          expect(response.identifiedClientState).toEqual({
+            clientId: ibcTest.clientId,
+            clientState: {
+              typeUrl: "/ibc.lightclients.tendermint.v1.ClientState",
+              value: jasmine.any(Uint8Array),
+            },
+          });
+
+          tmClient.disconnect();
+        });
+      });
 
-        const response = await client.ibc.unverified.packetCommitments(ibcTest.portId, ibcTest.channelId);
-        expect(response.commitments).toEqual([ibcTest.packetState]);
-        expect(response.pagination).toBeDefined();
-        expect(response.pagination).not.toBeNull();
-        expect(response.height).toBeDefined();
-        expect(response.height).not.toBeNull();
+      describe("consensusState", () => {
+        xit("works", async () => {
+          pendingWithoutSimapp();
+          const [client, tmClient] = await makeClientWithIbc(simapp.tendermintUrl);
+
+          const response = await client.ibc.unverified.channel.consensusState(
+            ibcTest.portId,
+            ibcTest.channelId,
+            // TODO: Find valid values
+            0,
+            0,
```

Comment:
I'm not sure which values I need here. Do I need to define them dynamically? @ethanfrey